IBPSA/Fluid/HeatPumps/ModularReversible/Examples/AirToWater2D_OneRoomRadiator.mo

```diff
@@ -87,6 +87,16 @@ equation
   heat pump model. Please check the associated documentation for
   further information.
 </p>
+<p>
+  Furthermore, this example is useful to understand the different sizing options.
+  The selected value of <code>QCoo_flow_nominal</code> leads to
+  conflicting scaling factors for heating and cooling operation. This triggers a warning.
+  Using the default <code>QCoo_flow_nominal</code>, the warning disappears. 
+  Increasing (or decreasing) it, the number of on-off switches will increase (or decrease)
+  in cooling operation due to the low inertia of the system. 
+  In real air-to-water systems, thermal energy storage would be used to 
+  decrease cycling if the device is oversized.
+</p>  
```

Comment:
I think it would be better to set up the model to avoid the warning
```
Warning: The following was detected at time: 0
  In AirToWater2D_OneRoomRadiator.heaPum: Scaling factors for heating and cooling
    operation differ by 41.2778 %. The simulated nominal heating and cooling
    capacities may not be realistic for a single device
  Failed condition: abs(heaPum.scaFacHea-heaPum.scaFacCoo)/heaPum.scaFacHea < heaPum.limWarSca
```
as from a user point of view, I would be concerned about seeing examples that don't seem to be configured in a realistic way.
Can you configure the model to avoid this warning.

If there is indeed a large discrepancy between loads, adding a small buffer tank in the return would help.

Reply:
Ok, I removed the warning by using the default and enabled the minimal runtime which was previously disabled. 
![image](https://github.com/ibpsa/modelica-ibpsa/assets/22442514/f2e33adb-323e-4580-a22a-2e7cfdfcf23c)
Now, each operation lasts at least 300 s.